frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx

```diff
@@ -330,7 +331,8 @@ const ChatWithInstructions: FC<
                   <div className="flex items-end justify-end gap-4">
                     <GeneralButton
                       text="I understand"
-                      onClick={() => setReadInstructions(true)}
+                      onClick={() => {setReadInstructions(true)}
```

Comment:
Andrew now that you removed the other function call please also remove the brackets before and after the function call as they are no longer needed. What it means is leave it as it was please:  
```javascript
onClick={() => setReadInstructions(true)}
```

Reply:
@shincap8 - okay, made this change